packages/transformed-erc20-extractor/src/get-event-data.js

```diff
@@ -0,0 +1,25 @@
+const { web3 } = require('@0x-event-extractor/shared');
+
+const getEventData = logEntry => {
+  const decoded = web3.getWrapper().abiDecoder.tryToDecodeLogOrNoop(logEntry);
+
+  const {
+    inputToken,
+    inputTokenAmount,
+    outputToken,
+    outputTokenAmount,
+    taker,
+  } = decoded.args;
+
+  const eventData = {
+    inputToken,
+    inputTokenAmount: inputTokenAmount.toString(),
```

Comment:
This is a BigNumber so it's being converted to a string for persistence in MongoDB.